Make.inc

```diff
@@ -207,75 +224,80 @@ ifneq (,$(findstring MSYS,$(BUILD_OS)))
 override BUILD_OS := WINNT
 endif
 
-ifeq ($(OS), WINNT)
-fPIC =
 ifeq ($(BUILD_OS), WINNT)
-PATH := $(PATH):$(build_libdir):$(build_private_libdir):/c/Program Files/7-zip
+PATH := $(PATH):$(build_libdir):$(build_private_libdir):/c/Program Files/7-zip:$(JULIAHOME)/dist-extras
```

Comment:
No, considering the only dll in either `build_libdir` or `build_private_libdir` is sys.dll I highly doubt these are doing anything. The addition of 7zip here is also quite old, and unpleasantly hardcoded - https://github.com/JuliaLang/julia/commit/c2323fbd25718707a4b9e2318d83ac16689da04b. Cygwin/msys2 git can act up if your path gets too long, so would be good to drop this.